packages/date-time-picker/src/vaadin-date-time-picker-mixin.js

```diff
@@ -528,7 +552,6 @@ export const DateTimePickerMixin = (superClass) =>
         this.datePlaceholder = newDatePicker.placeholder;
         this.initialPosition = newDatePicker.initialPosition;
         this.showWeekNumbers = newDatePicker.showWeekNumbers;
-        this.__syncI18n(this, newDatePicker, datePickerI18nProps);
```

Comment:
This looks quirky, it synchronized the i18n props from slotted elements into its own i18n. So it looks like either the custom I18N on DateTimePicker gets overridden by defaults from slotted elements, or the custom I18N from DateTimePicker overrides the custom I18N on slotted elements?

Maybe there is some more context on why this was added. But it seems safer to let one component be the source of truth, which now would be the DateTimePicker.